spec/restaurant_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'restaurant'
+
+describe Restaurant do
+
+  subject(:restaurant) { described_class.new }
+
+  context "#take_away service" do
+
+    it "shows menu when requested" do
+      expect(restaurant.read_menu).to eq ["Chips - £0.89", "Chicken Tikka - £3.99", "Burger - £2.99", "Veggie Wrap - £4.99"]
```

Comment:
Metrics/LineLength: Line is too long. [124/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)